oz/core/actions.py

```diff
@@ -65,6 +65,13 @@ def init(project_name):
 def server():
     """Runs the server"""
 
+    if oz.settings["logging"]:
```

Comment:
(I agree this is extremely poorly communicated, which is why we never noticed until now)